src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/import_multiple_feature_sections.json

```diff
@@ -45,4 +45,4 @@
       ]
     }
   }
-  }
```

Comment:
Can you revert the file? It only differs in whitespace

Reply:
Updated